# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
# Commit: a1c16b86d3a7
# Full Hash: a1c16b86d3a7f328a690c00f86b837a8a67a656c
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Remove RtcpEventObserver. r=ng
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D102287
# ==============================================================================

diff -r c7dbdd82e8e8 -r a1c16b86d3a7 dom/media/webrtc/libwebrtcglue/AudioConduit.cpp
--- a/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Mon Feb 01 09:28:23 2021 +0100
+++ b/dom/media/webrtc/libwebrtcglue/AudioConduit.cpp	Mon Jan 18 16:41:26 2021 +0100
@@ -197,12 +197,6 @@
       mDtmfPayloadType, mDtmfPayloadFrequency, eventCode, lengthMs);
 }
 
-void WebrtcAudioConduit::SetRtcpEventObserver(
-    mozilla::RtcpEventObserver* observer) {
-  MOZ_ASSERT(NS_IsMainThread());
-  mRtcpEventObserver = observer;
-}
-
 void WebrtcAudioConduit::GetRtpSources(
     nsTArray<dom::RTCRtpSourceEntry>& outSources) {
   MOZ_ASSERT(NS_IsMainThread());